src/hw2/Comics.java

```diff
@@ -0,0 +1,39 @@
+package hw2;
+
+public class Comics extends Library implements Artistic {
+    private String current_author;
+    private int current_year;
```

Comment:
according to conventions fields and methods should be named in camelcase

Reply:
Thanks. In HW3 I resolve all about code convention.